infrastructure/config/config.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import "os"
+
+// Config ...
+type Config struct {
+	AppPort           string
+	AppHost           string
+	AppEnv            string
+	PgHost            string
```

Comment:
Move to postgres file, it will make config file larger if there is another database

Reply:
atau untuk config postgresql nya dibikin struct gitu mas, isinya nanti configurasi si postgres nya?
apa gimana?